pom.xml

```
@@ -101,16 +101,44 @@
     <dependency>
       <groupId>org.kohsuke</groupId>
       <artifactId>github-api</artifactId>
-      <classifier>shaded</classifier>
-      <version>1.105</version>
+      <version>1.112</version>
     </dependency>
+    <dependency>
+      <groupId>org.javatuples</groupId>
+      <artifactId>javatuples</artifactId>
+      <version>1.2</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>annotations</artifactId>
+      <version>3.0.1u2</version>
+    </dependency>
```

The current preference seems to be that we use spotbugs annotations rather than findbugs annotations.  The findbugs project has been superseded by spotbugs.  The spotbugs project remains compatible with findbugs.
```suggestion
    <dependency>
      <groupId>com.github.spotbugs</groupId>
      <artifactId>spotbugs-annotations</artifactId>
      <exclusions>
        <exclusion>
          <groupId>com.google.code.findbugs</groupId>
          <artifactId>jsr305</artifactId>
        </exclusion>
      </exclusions>
      <optional>true</optional>
      <version>4.0.5</version>
    </dependency>
    <dependency>
```

---

This change was just moving the dependency from the bottom of the list of dependencies to a higher place near other, similar, dependencies. We can definitely switch to spotbugs, I'll verify in testing and update the PR.